packages/usa-combo-box/src/styles/_usa-combo-box.scss

```diff
@@ -68,7 +68,6 @@ $close: map-merge(
     // Placeholder overrides to ensure color contrast compliance accross browsers
     &::placeholder {
       opacity: 1;
-      color: color("disabled-dark");
```

Comment:
Browsers set the color for the placeholder via the `-webkit-text-fill-color` above. Removing this color definition because it seems unnecessary.